src/components/MessageArea.tsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect, useState, useCallback } from "react";
+
+const MOCK_CLIENT_ID = "1";
+const MOCK_USER_ID = "user-123";
+
+type Message = {
+  id: string;
+  content: string;
+  sentAt: string;
+  senderId: string;
+};
+
+const MessageArea = ({ initialMessages }: { initialMessages?: Message[] }) => {
+  const [messages, setMessages] = useState<Message[]>(initialMessages || []);
+
+  const token = localStorage.getItem("token");
+  const clientId = MOCK_CLIENT_ID;
+  const userId = MOCK_USER_ID;
+
+  const fetchMessages = useCallback(async () => {
+    if (!clientId || !token) return;
+    try {
+      const res = await fetch(
```

Comment:
This isn't actually displaying any messages for me, should I be seeing something? 
![image](https://github.com/user-attachments/assets/cddc8093-680f-4520-9657-266fdc258451)